src/Components/Search/NewSearch/NewSearchInputPills.tsx

```diff
@@ -169,6 +169,25 @@ const NewSearchInputPills: FC<NewSearchInputPillsProps> = ({
     }
   }
 
+  const handlePillClick = (pill: PillType) => {
+    onPillClick(pill)
+
+    // If user clicks on the leftmost or the rightmost pill, scroll to the left or to the right
+    // so that the pill is not hidden behind the chevron
+    const pillsContainer = pillsRef.current
+    if (pill == PILLS[0]) {
+      pillsContainer?.scrollBy({
+        left: -150, // Can be any number smaller than a pill width, 120 is the width of the current widest pill
+        behavior: "smooth",
+      })
+    } else if (pill == PILLS[PILLS.length - 1]) {
+      pillsContainer?.scrollBy({
+        left: 150, // Can be any number bigger than a pill width, 120 is the width of the current widest pill
+        behavior: "smooth",
+      })
+    }
+  }
```

Comment:
Just detect if you're scrolled over and fade out the gradient? This seems like the wrong solution here.